arangod/Aql/Query.cpp

```diff
@@ -1150,16 +1150,16 @@ uint64_t Query::calculateHash() const {
   // handle "fullCount" option. if this option is set, the query result will
   // be different to when it is not set!
   if (_queryOptions.fullCount) {
-    hashval = fasthash64(TRI_CHAR_LENGTH_PAIR("fullcount:true"), hashval);
+    hashval = fasthash64("fullcount:true", /*strlen*/ 14, hashval);
```

Comment:
I don't want to change the API of fasthash, because it is 3rd-party code. But I agree the hard-coded string lengths are bad.
So I will use some hard-coded constexpr string_views here, too.